src/nixos/machines/tupac/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+# shellcheck shell=sh # POSIX
+
+# Invokation: PATH_TO_THIS_REPO $ nix-shell -p zfs --run ./src/nixos/machines/tupac/deploy.sh
+
+export LC_ALL=C
+
+set -e # Exit on false return
+
+ragenixDir="/run/agenix"
+
+[ -d "$ragenixDir" ] || sudo mkdir "$ragenixDir"
+[ -d "/tmp/ragenix" ] || mkdir /tmp/ragenix
```

Comment:
Temporary files in `/tmp/ragenix` are not cleaned up after use, it could lead to security risks or clutter

Reply:
> https://github.com/NiXium-org/NiXium/pull/74#discussion_r1660036073 -- @TanvirOnGH (https://github.com/NiXium-org/NiXium/pull/74#discussion_r1660036073)

It's a work in progress implementation that is being tested in the wild before implemented in NiXium, Full implementation is in https://github.com/NiXium-org/NiXium/pull/78

I am aware of this and other issues, please review 78 when it's ready.